sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/models/AzureCliTokenTests.java

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation.models;
+
+import com.azure.json.JsonProviders;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonWriter;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class AzureCliTokenTests {
+
+    String jsonWithExpiresOnUnixTime = "{\n"
+        + "  \"accessToken\": \"tokenValue\",\n"
+        + "  \"expiresOn\": \"2024-02-28 12:05:53.000000\",\n"
+        + "  \"expires_on\": 1709150753,\n"
```

Comment:
Would there be a case where both are included in a return?

Reply:
Yes, the default case today is that `az` returns both. It can't remove the old one for backward compat reasons.